helm/sealed-secrets/values.yaml

```diff
@@ -105,6 +105,10 @@ tolerations: []
 ##
 updateStatus: true
 
+## @param keyrenewperiod Specifies key renewal period. Default 30 days
+##
+keyrenewperiod: "720h30m"
```

Comment:
By default, the parameter shouldn't be set so the flag isn't passed, relying on the controller binary default value:
```suggestion
## @param keyrenewperiod Specifies key renewal period. Default 30 days
## e.g
## keyrenewperiod: "720h30m"
##
keyrenewperiod: ""
```